python-cssselect2 fails to build with Python 3.9.0a2.

python-cssselect2 fails to build with Python 3.9.0a2.

    _____________ ERROR collecting cssselect2/tests/test_cssselect2.py _____________
    cssselect2/tests/test_cssselect2.py:48: in <module>
        TEST_DOCUMENT = get_test_document()
    cssselect2/tests/test_cssselect2.py:31: in get_test_document
        parent = next(e for e in document.getiterator() if e.get('id') == 'root')
    E   AttributeError: 'ElementTree' object has no attribute 'getiterator'

See https://docs.python.org/3.9/whatsnew/3.9.html#removed

"Methods getchildren() and getiterator() in the ElementTree module have been removed. They were deprecated in Python 3.2. Use functions list() and iter() instead. The xml.etree.cElementTree module has been removed." 

Downstream bug report: https://bugzilla.redhat.com/show_bug.cgi?id=1793518

--- a/cssselect2/tests/test_cssselect2.py
+++ b/cssselect2/tests/test_cssselect2.py
@@ -28,7 +28,12 @@ def load_json(filename):
 
 def get_test_document():
     document = etree.parse(resource('content.xhtml'))
-    parent = next(e for e in document.getiterator() if e.get('id') == 'root')
+    if hasattr(document, 'iter'):
+        iterator = document.iter()
+    else:
+        iterator = document.getiterator()
+
+    parent = next(e for e in iterator if e.get('id') == 'root')
 
     # Setup namespace tests
     for id in ('any-namespace', 'no-namespace'):

